engine/apps/metrics_exporter/tasks.py

```diff
@@ -133,9 +128,65 @@ def calculate_and_cache_metrics(organization_id, force=False):
     metric_alert_groups_total_key = get_metric_alert_groups_total_key(organization_id)
     metric_alert_groups_response_time_key = get_metric_alert_groups_response_time_key(organization_id)
 
+    recalculate_timeout = get_metrics_recalculation_timeout()
     metrics_cache_timeout = recalculate_timeout + TWO_HOURS
     cache.set(metric_alert_groups_total_key, metric_alert_group_total, timeout=metrics_cache_timeout)
     cache.set(metric_alert_groups_response_time_key, metric_alert_group_response_time, timeout=metrics_cache_timeout)
-    if metrics_cache_timer["forced_started"]:
+    if force:
+        metrics_cache_timer_key = get_metrics_cache_timer_key(organization_id)
+        metrics_cache_timer = cache.get(metrics_cache_timer_key)
         metrics_cache_timer["forced_started"] = False
-        cache.set(metrics_cache_timer_key, metrics_cache_timer, timeout=recalculate_timeout - ONE_HOUR)
+        cache.set(metrics_cache_timer_key, metrics_cache_timer, timeout=recalculate_timeout - TWO_HOURS)
```

Comment:
To be sure, what's the reason for the `ONE_HOUR` to `TWO_HOURS` change?

Reply:
Good catch, thanks! I was experimenting with the cache timer and commited this by mistake. In the current version there is no need to change the timeout